tests/test_particle_track.py

```diff
@@ -102,56 +95,54 @@ def test_steep():
 def test_steep_true():
     params.diff_coeff = 'invalid'
     params.steepest_descent = True
-    particle = particle_track.Particle(params)
+    particle = particle_track.Particles(params)
     assert particle.steepest_descent == True
 
 def test_steep_other():
     params.steepest_descent = 'other'
-    particle = particle_track.Particle(params)
+    particle = particle_track.Particles(params)
     assert particle.steepest_descent == False
 
 # testing of the run_iteration function
 def test_start_pairs_X():
+    particle = particle_track.Particles(params)
+    init_walk_data = particle.generate_particles(Np_tracer, seed_xloc,
```

Comment:
`generate_particles` no longer returns `init_walk_data`, so I imagine this throws an error. Shows up throughout this file.

Reply:
The beauty of the unit testing is that these don't throw errors as the tests are passing!

The underlying reason is that I think Python returns a `NoneType` object if no return is specified. But even so, I should probably remove those.